tools/rgprint/rgprint.cpp

```diff
@@ -0,0 +1,81 @@
+//
```

Comment:
License.

Reply:
Forgot about it. Actually it's just debugging tool, so I've also removed it from tools/CMakeLists.txt